### PR TITLE
[CIVIS-3137] Replace wall time documentation with new metric "total sessions duration"

### PR DESCRIPTION
**Pull request by TovRudyy**
### What does this PR do? What is the motivation?

Remove from our documentation mentions to "wall time" and replace it with the new metric "total sessions duration".

JIRA task: [CIVIS-3137](https://datadoghq.atlassian.net/browse/CIVIS-3137)

### Merge instructions

- [X] Please merge after reviewing

### Additional notes

[CIVIS-3137]: https://datadoghq.atlassian.net/browse/CIVIS-3137?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ A [flaky test][1] is a test that exhibit both a passing and failing status acros
 
 Flaky tests introduce risk and unpredictability into your CI system and end product. When people have to remember which tests are flaky, they lose trust in their test results, and a tremendous amount of time and resources are wasted on pipeline retries.
 
-For each branch, the list shows the number of new flaky tests introduced by the commit, the number of flaky commits, a comparison of the current wall time with the average default branch wall time, and the branch's latest commit details. 
+For each branch, the list shows the number of new flaky tests introduced by the commit, the number of flaky commits, total time of tests, and the branch's latest commit details.
```
Comment (urseberry) [inline content/en/continuous_integration/guides/flaky_test_management.md:25]:
```suggestion
For each branch, the list shows the number of new flaky tests introduced by the commit, the number of flaky commits, total test time, and the branch's latest commit details. 
```

File: content/en/continuous_integration/search/_index.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ You can access low-level and immediate insights:
 
 - See what tests are failing and why.
 - See your last commit's test results.
-- View the wall time of your tests in your feature branch and compare it to the default branch, to identify if you're about to introduce a performance regression.
+- View the total time duration of your tests in your feature branch.
```
Comment (urseberry) [inline content/en/continuous_integration/search/_index.md:105]:
```suggestion
- View the total duration of your tests in your feature branch.
```

File: content/en/continuous_integration/search/_index.md [hunk 3]
```diff
@@ -123,18 +123,14 @@ In this page, you can filter the list by name, test service, or commit SHA, or t
 
 #### Test results
 
-For each branch, you can see the test service, the number of failed, passed, and skipped tests, test regressions, wall time, the percentage of change compared to the default branch, when the commit was last updated, and the avatar of the author of the commit.
+For each branch, you can see the test service, the number of failed, passed, and skipped tests, test regressions, total time of tests, when the commit was last updated, and the avatar of the author of the commit.
```
Comment (urseberry) [inline content/en/continuous_integration/search/_index.md:126]:
```suggestion
For each branch, you can see the test service, the number of failed, passed, and skipped tests, test regressions, total test time, when the commit was last updated, and the avatar of the author of the commit.
```

File: content/en/continuous_integration/search/_index.md [hunk 4]
```diff
@@ -123,18 +123,14 @@ In this page, you can filter the list by name, test service, or commit SHA, or t
...
+
+Hovering over the commit author avatar shows detailed information about the latest commit.
```
Comment (urseberry) [inline content/en/continuous_integration/search/_index.md:128]:
```suggestion
Hovering over the commit author's avatar shows detailed information about the latest commit.
```

File: content/en/continuous_integration/troubleshooting.md [hunk 5]
```diff
@@ -93,28 +93,20 @@ If you can see test results data in the **Test Runs** tab, but not the **Tests**
 
 4. If no CI provider environment variables are found, tests results are sent with no Git metadata.
 
-### The tests wall time is empty
+### The tests total time is empty
```
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:96]:
```suggestion
### The total test time is empty
```

File: content/en/continuous_integration/troubleshooting.md [hunk 6]
```diff
@@ -93,28 +93,20 @@ If you can see test results data in the **Test Runs** tab, but not the **Tests**
...
+If you cannot see the tests total time it is likely that test suite level visibility is not enabled. To confirm this is the case, check if your language supports test suite level visibility in [Supported features][15]. If it is supported, update your tracer to the latest version.
```
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:97]:
```suggestion
If you cannot see the total test time, it is likely that test suite level visibility is not enabled. To confirm, check if your language supports test suite level visibility in [Supported features][15]. If test suite level visibility is supported, update your tracer to the latest version.
```

File: content/en/continuous_integration/troubleshooting.md [hunk 7]
```diff
@@ -93,28 +93,20 @@ If you can see test results data in the **Test Runs** tab, but not the **Tests**
...
 
-If you cannot see the tests wall time it is likely that the CI provider metadata is missing. To confirm this is the case, open a test execution in the [**Test Runs**][4] section, and check if the `ci.pipeline.id`, `ci.pipeline.name`, `ci.pipeline.number`, or `ci.job.url` tags are missing. If these tags are not populated, then nothing shows in the wall time column.
+If after updating the tracer version you still don't see the total time, contact [Datadog support][2] for help.
 
-1. Tracers use the environment variables set by the CI provider to collect this information. See [Running tests inside a container][7] for a list of environment variables that the tracer attempts to read for each supported CI provider. Make sure that the environment variables have the expected values set.
-2. Check that you are running your tests in a supported CI provider. For a list of supported CI providers, see [Running tests inside a container][7]. Only these CI providers can extract the information to enrich the test metadata with CI information.
-3. If you still don't see the wall time, contact [Datadog support][2] for help.
+### The tests tests total time is not what is expected
```
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:101]:
```suggestion
### The total test time is different than expected
```

File: content/en/continuous_integration/troubleshooting.md [hunk 8]
```diff
@@ -93,28 +93,20 @@ If you can see test results data in the **Test Runs** tab, but not the **Tests**
...
 
-### The tests wall time is not what is expected
-
-#### How wall time is calculated
-The wall time is defined as the time difference between the start time of the first test and the end time of the last test for the given pipeline.
+#### How total time is calculated
+The total time is defined as the total sum of the maximum test session durations.
```
Comment (urseberry) [inline content/en/continuous_integration/troubleshooting.md:104]:
```suggestion
The total time is defined as the sum of the maximum test session durations.
```

**Comment by TovRudyy** [comment]
Thank you @urseberry for the review 🙌

**Review by urseberry — APPROVED** [review]

**Comment by urseberry** [comment]
Is this ready to merge? If so, please check the "please merge after reviewing" checkbox in the description.

**Comment by TovRudyy** [comment]
> Is this ready to merge? If so, please check the "please merge after reviewing" checkbox in the description.

No. I plan to merge this the 23rd of November because it is blocked by a small backend change.

**Pull request merged**